### PR TITLE
chore: Token::getContent() phpdoc return type

### PR DESCRIPTION
**Pull request by mvorisek**
introduced wrongly in https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7598

Empty content is allowed and actually heavily used.

### CODE REVIEW

File: src/Tokenizer/CT.php [hunk 1]
```diff
@@ -74,6 +76,8 @@ public static function getName(int $value): string
 
         $tokens = self::getMapById();
 
+        \assert(isset($tokens[$value]));
```
Comment (keradus) [inline src/Tokenizer/CT.php:79]: @kubawerlos you were suggesting me better approaches for `asserts` few times. any idea here?

File: src/Tokenizer/Tokens.php [hunk 2]
```diff
@@ -94,7 +94,7 @@ class Tokens extends \SplFixedArray
      * was ever seen inside the collection (but may not be part of it any longer).
      * The key is token kind and the value is always true.
      *
-     * @var array<int|non-empty-string, int>
```
Comment (keradus) [inline src/Tokenizer/Tokens.php:97]: I am not confident that empty tokens should be registered here

File: src/Tokenizer/Token.php [hunk 3]
```diff
@@ -300,6 +300,8 @@ public static function getNameForId(int $id): ?string
 
         $name = token_name($id);
 
+        \assert('' !== $name);
```
Comment (keradus) [inline src/Tokenizer/Token.php:303]: can `token_name` ever return empty string ?

File: src/Tokenizer/Token.php [hunk 4]
```diff
@@ -45,7 +45,7 @@ final class Token
     private bool $changed = false;
 
     /**
-     * @param array{int, string}|string $token token prototype
+     * @param array{int, non-empty-string}|string $token token prototype
```
Comment (keradus) [inline src/Tokenizer/Token.php:48]: funny we provide empty string only in `Tokens::clearAt()`

File: src/Tokenizer/Token.php (hunk 3 above)
Comment (keradus) [reply src/Tokenizer/Token.php:303]: maybe let's convert to phpstan exception and wait remove exception when https://github.com/phpstan/phpstan/issues/11808 is solved

File: src/Tokenizer/Token.php (hunk 4 above)
Comment (mvorisek) [reply src/Tokenizer/Token.php:48]: good improvement!

Comment (mvorisek) [reply src/Tokenizer/Token.php:48]: but please fix the new phpstan errors as well

Comment (keradus) [reply src/Tokenizer/Token.php:48]: let's take as separated PR then

Comment (keradus) [reply src/Tokenizer/Token.php]: https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/8242

**Comment by keradus** [comment]
thank you @mvorisek

File: src/Tokenizer/Token.php (hunk 3 above)
Comment (keradus) [reply src/Tokenizer/Token.php:303]: https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/8245

**Pull request merged**